src/index.css

```diff
@@ -28,16 +28,18 @@ header h1 {
   font-size: 34px;
 }
 
-.board-list button {
+.refresh {
   height: 30px;
   font-size: 20px;
   width: 140px;
   padding: 3px;
   margin: 35px;
   justify-content: center;
+  border: 2px solid black;
 }
 
 .add form {
+  margin-top: 45px;
   display: flex;
   flex-direction: column;
   gap: 20px;
```

Comment:
- Kindly, move the refresh button to the right, minimize the submit button as a wireframe, and minimize the space between the form and the table to stick the wireframe and have a better UI 👍 

![Screenshot 2023-06-28 130237](https://github.com/tajulafreen/Leaderboard/assets/52010789/2500a93a-2ba9-4f2d-85bd-5c03693c7374)